### DISCUSSION TITLE
Power and consumption reporting from Bituo Technik SPM01-U01

### DISCUSSION THREAD

**Discussion by fishter** [Koenkk/zigbee2mqtt#24510] [Devices]
I have a couple of Bituo Technik SPM01-U01 devices (https://www.zigbee2mqtt.io/devices/SPM01-U01.html#bituo%2520technik-spm01-u01)

They are reporting the consumption correctly in kWh, but the instantaneous power reporting is out by a factor of 1000.

Power seems to be in kW and Apparent Power appears to be in kVA. The units used in Z2M are W and VA.

What debug can I provide to help get this converted correctly in Z2M? (I'm running the edge version 1.40.2-dev commit: [2b460d3])
I found a previous discussion (#20593) where I think some changes may have been added related to my issue.

**Comment by Patricen1** [comment]
Same issue there.

**Comment by theveninjeremy** [comment]
I just discover this issue on my SPM02X001. 
Does a fix exist?
